src/components/App/App.js

```diff
@@ -9,7 +9,9 @@ export default class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: []
+      movies: [],
+      filteredMovies: [],
```

Comment:
nice i like how you set the state as an empty array so we don't have errors running